modules/installation/proc-configuring-an-rhdh-instance-with-tls-in-kubernetes.adoc

```diff
@@ -0,0 +1,104 @@
+
+.Procedure
+
+. Enable the Kubernetes plugins in the `dynamic-plugins-rhdh.yaml` ConfigMap file:
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/backstage-plugin-kubernetes-backend-dynamic
+        disabled: false <1>
+      - package: ./dynamic-plugins/dist/backstage-plugin-kubernetes
+        disabled: false <2>
+        # ...  
+----
+<1> Set the value to `false` to enable the `backstage-plugin-kubernetes-backend-dynamic` plugin
+<2> Set the value to `false` to enable the `backstage-plugin-kubernetes` plugin.
+
+. Set the values in the kubernetes section of the `app-config-rhdh.yaml` ConfigMap file and configure the catalog sync options:
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+  # ...
+  catalog:
+    rules:
+      - allow: [Component, System, API, Resource, Location]
+    providers:
+      kubernetes:
+        openshift:
+          cluster: openshift
+          processor:
+            namespaceOverride: default
+            defaultOwner: guests
+          schedule:
+            frequency:
+              seconds: 30
+            timeout:
+              seconds: 5
+  kubernetes:
+    serviceLocatorMethod:
+      type: 'multiTenant'
+    clusterLocatorMethods:
+      - type: 'config'
+        clusters:
+          - url: <target-cluster-api-server-url> <1>
+            name: openshift
+            authProvider: 'serviceAccount'
+            skipTLSVerify: false <2>
+            skipMetricsLookup: true
+            dashboardUrl: <target-cluster-console-url> <3>
+            dashboardApp: openshift
+            serviceAccountToken: <service-account-token> <4>
+            caData: <service-ca-base64-encoded> <5>
+            # ...
+----
+<1> The base URL to the Kubernetes control plane. You can run the `kubectl cluster-info` command to get the base URL.
+<2>  Set the value of this parameter to `false` to enable the verification of the TLS certificate.
+<3> Optional: The link to the Kubernetes dashboard managing the ARO cluster. 
+<4>  Optional: The service account token for using the `serviceAccount` authentication provider.
+<5> Set the value of the parameter to `service-ca-base64-encoded` data, which is a base64-encoded certificate authority bundle in the .PEM format. 
+
+. Save the configuration changes.
+
+.Verification
+
+. Run the RHDH application to import your catalog:
++
+[source,terminal]
+----
+kubectl -n rhdh-operator get pods -w
+----
+
+. Verify that the pod log shows no errors for your configuration.
+. Navigate to your catalog entity to view the resources that you created in a Kubernetes tab.
```

Comment:
did we check this in the ARO cluster UI? do we have any command that we can run using CLI to navigate to this catalog entity?
This is needed so that we can make the verification procedure CLI-specific, rather than performing some steps with CLI and others with web UI.

Reply:
This is intended as checking the Entity page from the Catalog in the Developer Hub portal to ensure that the connection with the cluster is established successfully, the resources are read correctly and shown in DH, not the ARO cluster. So I would say there's no cli alternative.